assets/css/less/front-end/abstracts/variables/typography.less

```diff
@@ -6,6 +6,7 @@
 
 // Font Sizes
 @font-size-alert: 14px;
+@font-size-form-field-with-error: 12px;
```

Comment:
Maybe we can use something more generic here to make them easier to reuse? Like @font-size-sm and $font-size-xs?

Reply:
It would be nice to update @font-size-alert also to something more easily reusable. like this:
```
@font-size-sm: 14px;
@font-size-xs: 12px;
```